### PR TITLE
Fix readset and file filter based on release status

### PR DESCRIPTION
**Pull request by nafiz1001**

### CODE REVIEW

**Review by samouzegar10 — COMMENTED** [review]
seems good to me

**Review by UlysseFG — COMMENTED** [review]
Looks good overall ... the actual result should be tested, but this looks simpler than fetching metrics specifically.

File: triton-server/src/freezeman/api.ts [hunk 1]
```diff
@@ -129,53 +129,30 @@ export const getAuthenticatedAPI = (axios: AxiosInstance) => {
...
+                    `${LIMS_API_URL}/dataset-files/?readset__dataset__id=${id}&readset__release_status=${RELEASED}&limit=100000`,
                 )
             },
         },
         Readset: {
             listByDatasetId: async (
                 datasetId: Dataset["id"],
-            ): Promise<ListResponse<Readset>> => {
+            ): Promise<ListResponse<ReadsetWithMetrics>> => {
+                const RELEASED: ReleaseFlagReleased = 1
```
[the remark this answers is not in the document]
Comment (nafiz1001) [reply triton-server/src/freezeman/api.ts:145]: Yeah sadly typescript does not let you have constant values in a type declaration file.

[the remark this answers is not in the document]
Comment (nafiz1001) [reply triton-server/src/freezeman/api.ts:145]: Although I probably could have defined the constant in triton-server instead.

**Pull request merged**